rosdep/python.yaml

```diff
@@ -6620,6 +6620,16 @@ python3-meshio:
   arch: [python-meshio]
   debian: [python3-meshio]
   ubuntu: [python3-meshio]
+python3-mlflow-pip:
+  debian:
+    pip:
+      packages: [mlflow]
+  fedora:
+    pip:
+      packages: [mlflow]
+  ubuntu: 
+    pip:
+      packages: [mlflow]
```

Comment:
@n-meunier mind to add an entry for OSX and Gentoo? BTW it appears there's `mlflow` 1.11.0 [on OpenSUSE](https://build.opensuse.org/package/show/home:jayvdb:py-new/python-mlflow).

Reply:
Thanks for the review. I'm gonna add an entry for OSX and OpenSUSE. 
For Gentoo, I could not find the package in https://packages.gentoo.org/. Isn't it the condition for this entry?

Reply:
Yeah, it doesn't look like we can use anything but `pip` on Gentoo.

Reply:
I haven't found any other example with:
```
  gentoo:
    pip:
      packages: 
```
But I added it following your comment.

Reply:
My impression is that gentoo doesn't like alternative installers like pip. @allenh1  Can you confirm?

Reply:
> My impression is that gentoo doesn't like alternative installers like pip.

Yeah, you shouldn't install things with pip on gentoo. These days, if you attempt to run pip install you get an error message.

```
ERROR: (Gentoo) Please run pip with the --user option to avoid breaking python-exec
```

Previously it would just break your package manager :upside_down_face:.

What we can do, though, is create a package in the overlay repo. We've done this for a few deps. I'll make an issue there for it.

Reply:
I made a ticket to release the dependency [here](https://github.com/ros/ros-overlay/issues/1019). In the meantime, feel free to merge without a gentoo resolution, as there unfortunately isn't one.

Reply:
Thank you for the ticket and your explanation.